s3/src/main/scala/com/meltwater/fawn/s3/payloads.scala

```diff
@@ -0,0 +1,152 @@
+package com.meltwater.fawn.s3
+
+import cats.implicits._
+import com.lucidchart.open.xtract.XmlReader._
+import com.lucidchart.open.xtract.{__, XmlReader}
+import org.http4s.Headers
+
+/** Represents a generic response that contains the AWS Request Id and all remaining headers.
+  * @param requestId
+  *   Amazon AWS Request ID
+  * @param headers
+  *   All Response Headers
+  */
+case class GenericResponse(requestId: String, headers: Headers)
```

Comment:
what value does this provide vs `List[Headers]`? is it common to use the `requestId`? how many other headers can we get?